spec/features/posts/flat_spec.rb

```diff
@@ -9,7 +9,7 @@
 
     visit post_path(post, view: 'flat')
     expect(page).to have_selector('.post-container', count: 1)
-    expect(page).to have_selector('.post-post', count: 1)
+    expect(page).to have_selector('.post-reply', count: 1)
```

Comment:
This seems like a fair change, but I want to note that it _is_ different from posts#show, and we should maybe look into renaming things appropriately.